avahi-monitor.py

```diff
@@ -36,6 +38,16 @@
     # "_nfd._tcp",
     "_nfd._udp"]
 
+def face_uri_from_triplet(type, host, port):
+    if type == '_nfd._udp':
+        proto = 'udp4'
+    else:
+        proto = 'tcp4'
+    return "%s://%s:%s"%(proto, host, port)
```

Comment:
Nitpick: Is the `//` necessary here? Technically, the scheme and address of a URI only have to be separated by a colon (`:`).